ligand/models.py

```diff
@@ -311,188 +339,280 @@ def __str__(self):
+    log_bias_factor_c = models.CharField(max_length=60, null=True)
+    log_bias_factor_d = models.CharField(max_length=60, null=True)
+    log_bias_factor_e = models.CharField(max_length=60, null=True)
+    t_factor = models.CharField(max_length=60, null=True)
+    assay_description = models.CharField(max_length=900, null=True)
+    reference_ligand_id = models.CharField(max_length=60, null=True)
+    emax_ligand_reference = models.ForeignKey(Ligand, related_name='ExperimentAssay.bias_ligand_reference+',
+                                              on_delete=models.CASCADE,
+                                              null=True, blank=True)
+
+
 class BiasedPathways(models.Model):
     submission_author = models.CharField(max_length=50)
-    ligand = models.ForeignKey(Ligand, on_delete = models.CASCADE)
-    ligand_pubchem = models.CharField(max_length=40,null = True)
-    publication = models.ForeignKey(Publication, on_delete = models.CASCADE)
-    receptor = models.ForeignKey('protein.Protein', on_delete = models.CASCADE)
-    chembl = models.CharField(max_length=40,null = True)
-    relevance = models.CharField(max_length=50,null = True)
-    signalling_protein = models.CharField(max_length=20,null = True)
+    ligand = models.ForeignKey(Ligand, on_delete=models.CASCADE)
+    # lignad_pubchem = models.CharField(max_length=40, null=True)
+    publication = models.ForeignKey(Publication, on_delete=models.CASCADE)
+    receptor = models.ForeignKey('protein.Protein', on_delete=models.CASCADE)
+    chembl = models.CharField(max_length=40, null=True)
+    relevance = models.CharField(max_length=50, null=True)
+    signalling_protein = models.CharField(max_length=20, null=True)
+
 
 class BiasedPathwaysAssay(models.Model):
     biased_pathway = models.ForeignKey(
-                        BiasedPathways, related_name='biased_pathway',
-                        on_delete = models.CASCADE, null = True
-                        )
+        BiasedPathways, related_name='biased_pathway',
+        on_delete=models.CASCADE, null=True
+    )
     pathway_outcome_high = models.CharField(max_length=200)
-    pathway_outcome_summary = models.CharField(max_length=200, null = True)
-    pathway_outcome_detail  = models.CharField(max_length=200, null = True)
-    experiment_pathway_distinction = models.CharField(max_length=200, null = True)
-    experiment_system = models.CharField(max_length=40, null = True)
-    experiment_outcome_method= models.CharField(max_length=200, null = True)
+    pathway_outcome_summary = models.CharField(max_length=200, null=True)
+    pathway_outcome_detail = models.CharField(max_length=200, null=True)
+    experiment_pathway_distinction = models.CharField(
+        max_length=200, null=True)
+    experiment_system = models.CharField(max_length=40, null=True)
+    experiment_outcome_method = models.CharField(max_length=200, null=True)
+
 
+# Pathways Part - end
+
+
+#New ligand models
```

Comment:
Could you create a separate branch for the new ligand models?

Reply:
Done